<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>969</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自由视点视频播放系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/nbu.png</normaloff>Resources/nbu.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QPushButton" name="Button_RePlay">
          <property name="text">
           <string>回放</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="Button_ResetToEnd">
          <property name="text">
           <string>视频结尾</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Button_Play">
          <property name="text">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="Button_ResetToStart">
          <property name="text">
           <string>视频复位</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="Button_MultiStepBackward">
          <property name="text">
           <string>多帧步退</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="Button_StepForward">
          <property name="text">
           <string>单帧步进</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="Button_MultiStepForward">
          <property name="text">
           <string>多帧步进</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="Button_StepBackward">
          <property name="text">
           <string>单帧步退</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="Box_View">
        <property name="title">
         <string>视点选择</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_View1">
           <property name="text">
            <string>View 1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_View2">
           <property name="text">
            <string>View 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_View3">
           <property name="text">
            <string>View 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_View4">
           <property name="text">
            <string>View 4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_View5">
           <property name="text">
            <string>View 5</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Box_Effect">
        <property name="title">
         <string>特效选择</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="3">
          <widget class="QRadioButton" name="radioButton_Erode">
           <property name="text">
            <string>腐蚀</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton_None">
           <property name="text">
            <string>无</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="radioButton_Dilate">
           <property name="text">
            <string>膨胀</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButton_Gray">
           <property name="text">
            <string>灰度</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButton_Histogram">
           <property name="text">
            <string>直方图均衡</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="radioButton_EdgeDetect">
           <property name="text">
            <string>边缘检测</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QRadioButton" name="radioButton_WaterShed">
           <property name="text">
            <string>分水岭分割</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>graphicsView</zorder>
   <zorder>progressBar</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="MENU_OPEN">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="SUBMENU_OPEN"/>
   </widget>
   <widget class="QMenu" name="MENU_PLAY">
    <property name="title">
     <string>播放</string>
    </property>
    <addaction name="SUBMENU_SET"/>
    <addaction name="SUBMENU_CAPTURE"/>
   </widget>
   <widget class="QMenu" name="MENU_ABOUT">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="SUBMENU_ABOUT"/>
   </widget>
   <addaction name="MENU_OPEN"/>
   <addaction name="MENU_PLAY"/>
   <addaction name="MENU_ABOUT"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="SUBMENU_SET">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="SUBMENU_OPEN">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="SUBMENU_CAPTURE">
   <property name="text">
    <string>抓图</string>
   </property>
  </action>
  <action name="SUBMENU_ABOUT">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
